Lib/pydoc.py

```diff
@@ -1254,6 +1254,17 @@ def makename(c, m=object.__module__):
                 push('    ' + makename(base))
             push('')
 
+        # List the built-in subclasses, if any:
+        subclasses = [
+            cls.__name__ for cls in object.__subclasses__()
+            if cls.__module__ == 'builtins'
```

Comment:
Elsewhere in this file we test against `object.__module__` - I think this is relevant for python 2, but consistency seems useful anyway.

Reply:
Ah nevermind - this is not the builtin `object`.